### ISSUE TITLE
Mod Incompatability with Clay Solders Mod

### ISSUE THREAD

**Issue by MikoSphinx** [ForgeEssentials/ForgeEssentials#2111]
https://www.dropbox.com/s/wuu1lvc1uayf37t/fml-server-latest.log?dl=0 
When upgrading from 1182 to 1187, we started getting this crash. Deleting permissions file does nothing to solve it.
https://www.dropbox.com/s/ug3518juc7t1cl6/permissions.json?dl=0
permission file report too.

**Comment by spacebuilder2020** [comment]
So what was the issue?

**Comment by MikoSphinx** [comment]
Found that Clay Soldiers mod doesn't like FE, so had to take out the clay soldiers mod for the server not to crash each load up.

**Comment by spacebuilder2020** [comment]
Sounds like a mod compatibility bug.  Can you confirm that the same version of clay solders worked fine in 1182?

**Comment by spacebuilder2020** [comment]
@MikoSphinx Did clay solders work fine with 1182?

**Comment by MikoSphinx** [comment]
We have not tried in that version but decided not to have it in the modpack right now.

**Comment by MikoSphinx** [comment]
We just had a recent crash on the server though  https://www.dropbox.com/s/rardnu8hi8hcxmq/Crash%20Report%207.19.16.txt?dl=0 trying to fix it at the moment.

**Comment by WWDragon** [comment]
Original issue WAS clay soldiers.

Current issue is the npc dulplicates bug in permissions.

**Comment by spacebuilder2020** [comment]
We are still working on the duplicate bug.  @olee Did not like the first fix I made cause it was too aggressive, so we are working on a less intrusive fix.  See #2103 and https://github.com/ForgeEssentials/ForgeEssentials/commit/668592376b00fbde9968e18fa4dd12e0a2387276.  As for your original crash report, it looks like it was caused by a conflicting mixin, but no mixins were changed in between the two updates.
